homeassistant/components/pvoutput/sensor.py

```diff
@@ -59,7 +60,10 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
     method = "GET"
     payload = auth = None
     verify_ssl = DEFAULT_VERIFY_SSL
-    headers = {"X-Pvoutput-Apikey": api_key, "X-Pvoutput-SystemId": system_id}
+    headers = {
+        "X-Pvoutput-Apikey": Template(api_key, hass),
```

Comment:
This is also not the right fix. This means that anything consuming `RestData` is now broken. This is a breaking change inside the Rest integration that needs to be fixed.

Reply:
This is part of the pvoutput integration cannot see how this fix breaks the rest integration.

Reply:
Fixed RestData so this fix is no longer needed. https://github.com/home-assistant/core/pull/61296